projects/storefrontlib/cms-components/content/pdf/pdf.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="data$ | async as data">
+  <div class="pdf-container">
+    <a [ngClass]="url ? 'active' : 'disabled'" [href]="url" target="_blank">
+      {{ data?.name || data?.pdfFile?.altText || 'document.pdf' }}
```

Comment:
Can you verify with UX/UI whether we should append .pdf to the name of the file? Easier for people to know its a pdf file but, I noticed other websites don't include the .pdf extension in the name